js/menu.js

```diff
@@ -0,0 +1,14 @@
+const btn = document.querySelector('.mb-menu');
+const menu = document.querySelector('.hamburger');
+const menuList = document.querySelectorAll('.mb-menu-list a');
+
+btn.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  document.body.classList.toggle('no-scroll');
+});
+
+menuList.forEach((list) => {
+  list.addEventListener('click', () => {
+    menu.classList.remove('active');
+  });
+});
```

Comment:
hey Robert, Thanks for the feedback
, I ran the same checks on my chrome and edge browser and none of the above-mentioned changes are seen on my local browser, there is no gap or issue of scrolling, please if you can just tell me how to reproduce this error on my local machine... Thank you.
![image](https://user-images.githubusercontent.com/81693905/156225645-bede3de0-d3a3-4ed7-ae62-84647d0eeb2f.png)

Reply:
Hello, I ran into that bug by setting a view with a width lower than 300px on my screen.

Reply:
![image](https://user-images.githubusercontent.com/81693905/156228826-a722d5bf-26b5-44f4-8943-26b314aab584.png)
Hello, please assist me here because i checked down to 200px its scrolling well sir,

Reply:
This is weird. It might be because the resolution of the screen itself. I tested it on a FullHD (1920x1080) screen, is yours perhaps a smaller size?

Reply:
on the above size the hamburger menu won't show

Reply:
My bad, I'm sorry @EzekielUtshudi , the bug occurs after you open the menu, click on an element and then open the menu again. This second time, the menu will allow the user to scroll the page, and you'll see the content behind scrolling like in the screenshot.

Reply:
It's alright, since Monday I have been working on this project, please I NEED YOUR ASSISTANCE, how to solve this ? because I still can't see it ?

Reply:
I've attached a GIF of the interactions that trigger the error.

![](https://i.imgur.com/BPAabAx.gif)

Reply:
I added more descriptive comments and the implemented changes